src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -189,6 +199,44 @@ class HuiShoppingListCard extends SubscribeMixin(LitElement)
     `;
   }
 
+  private _renderItems(items: ShoppingListItem[]) {
+    return html`
+      ${repeat(
+        items,
+        (item) => item.id,
+        (item) =>
+          html`
+            <div class="editRow" item-id=${item.id}>
```

Comment:
```suggestion
            <div class="editRow" .itemId=${item.id}>
```

Reply:
actually do you need this

Reply:
> actually do you need this

@zsarnett  Yes, [`Sortable` uses it](https://github.com/home-assistant/frontend/pull/7296/files#diff-51b956cdb853a2c20ff93c5cd7d3963fR322), so that we can conveniently [get reordered item ids array](https://github.com/home-assistant/frontend/pull/7296/files#diff-51b956cdb853a2c20ff93c5cd7d3963fR325).

Reply:
Well we actually could avoid this by mapping reordered `this._uncheckedItems` to item ids after [this](https://github.com/home-assistant/frontend/pull/7296/files#diff-51b956cdb853a2c20ff93c5cd7d3963fR333).

Which way do you think is better?

Reply:
What you are doing is fine. I forgot about that for sortable.